src/map_saver.cpp

```diff
@@ -61,10 +62,29 @@ bool MapSaver::saveMapCallback(
     RCLCPP_INFO(node_->get_logger(),
       "SlamToolbox: Saving map as %s.", name.c_str());
     int rc = system(("ros2 run nav2_map_server map_saver_cli -f " + name  + " --ros-args -p map_subscribe_transient_local:=true").c_str());
+    if (rc == 0) {
+      response->result = response->RESULT_SUCCESS;
+    }
+    else if (rc == 64000) {
```

Comment:
It is not always going to be 64000 for a segmentation fault, often I see -11 or any number of other values. This seems fragile, I think keep this as `RESULT_UNDEFINED_FAILURE`